### PR TITLE
for sonet.json

### PR DESCRIPTION
**Pull request by codersonet**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel or Netlify.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.

### CODE REVIEW

**Comment by CoderSoNet** [comment]
I have created sonet.json for emailing through website.

File: domains/sonet.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  "owner": {
+    "username": "CoderSoNet",
+    "email": "support@sonet.is-a.dev",
+    "twitter": "ShanuUniyal_On",
+    "discord": "781127625263415326"
+  },
+  "record": {
+    "TXT": "v=spf1 a mx include:_spf.elasticemail.com include:_spf.google.com ~all",
+    "DKIM": {"api._domainkey": "k=rsa;t=s;p=MIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQCbmGbQMzYeMvxwtNQoXN0waGYaciuKx8mtMh5czguT4EZlJXuCt6V+l56mmt3t68FEX5JJ0q4ijG71BGoFRkl87uJi7LrQt1ZZmZCvrEII0YO4mp8sDLXC8g1aUAoi8TJgxq2MJqCaMyj5kAm3Fdy2tzftPCV/lbdiJqmBnWKjtwIDAQAB"},
+    "CNAME": {"tracking": "api.elasticemail.com"}
```
Comment (WilliamDavidHarrison) [inline domains/sonet.json:13]: If this is meant to be on `tracking.sonet.is-a.dev` you'll need a seperate JSON file for it.

File: domains/sonet.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  "repo": "https://github.com/CoderSoNet/codersonetwebsite.github.io",
+  "owner": {
+    "username": "CoderSoNet",
+    "email": "support@sonet.is-a.dev",
+    "twitter": "ShanuUniyal_On",
+    "discord": "781127625263415326"
+  },
+  "record": {
+    "TXT": "v=spf1 a mx include:_spf.elasticemail.com include:_spf.google.com ~all",
+    "DKIM": {"api._domainkey": "k=rsa;t=s;p=MIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQCbmGbQMzYeMvxwtNQoXN0waGYaciuKx8mtMh5czguT4EZlJXuCt6V+l56mmt3t68FEX5JJ0q4ijG71BGoFRkl87uJi7LrQt1ZZmZCvrEII0YO4mp8sDLXC8g1aUAoi8TJgxq2MJqCaMyj5kAm3Fdy2tzftPCV/lbdiJqmBnWKjtwIDAQAB"},
```
Comment (WilliamDavidHarrison) [inline domains/sonet.json:12]: Invalid record.

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/sonet.json (hunk 2 above)
Comment (CoderSoNet) [reply domains/sonet.json:12]: Then how can I make domain for emailing 😭

Comment (WilliamDavidHarrison) [reply domains/sonet.json:12]: > Then how can I make domain for emailing 😭

Use MX records...

**Review by wdhdev — APPROVED** [review]

**Pull request merged**